test/fixtures/generateEndorseCertificateFormModel.ts

```diff
@@ -0,0 +1,7 @@
+import EndorseCertificateFormModel from 'app/models/form/endorseCertificateFormModel'
```

Comment:
The file should be named endorseCertificateFormModel.fixtures.ts. The name of the class is ok